scripts/dedup_across_files.py

```diff
@@ -0,0 +1,11 @@
+seen = set()
+seen_add = seen.add
+
+for filename in ['word.csv', 'trending.csv', 'fixed_expressions.csv', 'phrase_fragment.csv']:
```

Comment:
@hfhchan The priority list is here, feel free to change the order if you are not comfortable with it

Reply:
`'fixed_expressions.csv', 'phrase_fragment.csv', 'trending.csv', 'word.csv'` thanks